policy/modules/system/systemd.if

```diff
@@ -2290,16 +2292,16 @@ interface(`systemd_map_resolved_exec_files',`
 
 ########################################
 ## <summary>
-##	Send and receive messages from
-##	systemd resolved over dbus.
+##	Exchange messages with
+##	systemd resolved over dbus or varlink.
 ## </summary>
 ## <param name="domain">
 ##	<summary>
 ##	Domain allowed access.
 ##	</summary>
 ## </param>
 #
-interface(`systemd_dbus_chat_resolved',`
+interface(`systemd_chat_resolved',`
```

Comment:
All interfaces in the policy currently have "dbus_chat" as a part of the name.

Reply:
Yeah, but now it's either dbus or varlink. How should it be called in that case?

Reply:
I see, so just need to adjust to it.